tests/dapr_tests.mk

```diff
@@ -445,6 +446,7 @@ setup-test-components: setup-app-configurations
 	$(KUBECTL) apply -f ./tests/config/dapr_vault_secretstore.yaml --namespace $(DAPR_TEST_NAMESPACE)
 	$(KUBECTL) apply -f ./tests/config/uppercase.yaml --namespace $(DAPR_TEST_NAMESPACE)
 	$(KUBECTL) apply -f ./tests/config/pipeline.yaml --namespace $(DAPR_TEST_NAMESPACE)
+	$(KUBECTL) apply -f ./tests/config/pluggable_state.yaml --namespace $(DAPR_TEST_NAMESPACE)
```

Comment:
Call it dapr_redis_pluggable_state.yaml

Reply:
I guess your idea is to use the DAPR_TEST_STATE_STORE variable to switch between cosmos/redis/redis_pluggable is that correct?

If yes, wouldn't better to have both running at same time and switch between cosmos and redis as it is currently?

if I got it wrong, just ignore this comment.